Description: enable generation of full manpage
 The upstream release apparently misses the full manpage, which is to be
 included in all releases as stated on the upstreams' homepage. This patch
 enables the manpage generation and removes unneeded build instructions.
Author: Simon Elsbrock
Forwarded: no, manpage to be included in next release tarball
---
Index: jq/Makefile.am
===================================================================
--- jq.orig/Makefile.am	2013-05-19 10:48:18.000000000 +0000
+++ jq/Makefile.am	2014-02-17 14:37:54.526939179 +0000
@@ -52,27 +52,16 @@
 # Thus, making "dist" depend on "real_docs" ensures that we never build a tarball with
 # a stub manpage.
 
-man_MANS = jq.1
+#man_MANS = jq.1
 .PHONY: real_docs
-if ENABLE_DOCS
 jq.1: $(srcdir)/docs/content/3.manual/manual.yml
-	( cd ${abs_srcdir}/docs; '$(BUNDLER)' exec rake manpage ) > $@ || { rm -f $@; false; }
+	( cd ${abs_srcdir}/docs; exec rake manpage ) > $@ || { rm -f $@; false; }
 jq.1.default: $(srcdir)/docs/default_manpage.md
-	( cd ${abs_srcdir}/docs; '$(BUNDLER)' exec rake manpage_default ) > $@ || { rm -f $@; false; }
+	( cd ${abs_srcdir}/docs; exec rake manpage_default ) > $@ || { rm -f $@; false; }
 real_docs: jq.1
 	if cmp jq.1 $(srcdir)/jq.1.default > /dev/null; then\
 	  rm -f jq.1; $(MAKE) $(AM_MAKEFLAGS) jq.1;\
 	fi
-else
-jq.1: $(srcdir)/jq.1.default
-	cp $^ $@
-real_docs:
-	@echo "Ruby dependencies not found, cannot build manpage." > /dev/stderr
-	@echo "Follow the instructions in docs/README.md to install them" > /dev/stderr
-	@echo "and then rerun ./configure" > /dev/stderr
-	false
-endif
-
 
 ### Packaging
 
@@ -100,7 +89,7 @@
 # README.md is expected in Github projects, good stuff in it, so we'll 
 # distribute it and install it with the package in the doc directory.
 docdir = ${datadir}/doc/${PACKAGE}
-dist_doc_DATA = README.md COPYING AUTHORS README
+dist_doc_DATA = README.md AUTHORS
 
 releasetag:
 	git tag -s "jq-$$(cat VERSION)" -m "jq release $$(cat VERSION)"
